app/presenters/brexit_taxons_presenter.rb

```diff
@@ -10,9 +10,9 @@ class BrexitTaxonsPresenter
 
   def featured_taxons
     @featured_taxons ||= FEATURED_TAXONS
-      .map { |base_path| Taxon.find(base_path) }
-      .map
-      .with_index(1) { |taxon, index| BrexitTaxonPresenter.new(taxon, index) }
+      .map { |base_path| ContentItem.find!(base_path) }
+      .each_with_index
```

Comment:
I believe @vanitabarrett used `with_index(1)` rather than `each_with_index` because the index needs to start at 1 rather than zero.